imports/plugins/core/layout/client/templates/layout/footer/footer.html

```diff
@@ -1,7 +1,88 @@
+                <div class="" id="footer-links">
+                    <div class="footer-links-body">
+											<h4 class="text-center">GET TO KNOW US</h4>
+											<div class="container">
+												<ul class>
+													<li>About us</li>
+													<li>Contact us</li>
+													<li>Terms and Conditions</li>
+													<li>Policy</li>	
+												</ul>
+											</div>
+                      <br>
+											<h4 class="text-center">MAKE MONEY WITH US</h4>
+											<div class="container">
+												<ul>
+													<li>Sell on Reaction</li>
+													<li>Become an Affiliate Partner</li>
+													<li>Become a Sales Consultant</li>
+												</ul>
+											</div>
+                    </div>
+                  </div>
+            </div>
+            <div class="col-md-4 col-lg-4 container">
+              <div class="footer-newsletter-body">
+                <div id="footer-newsletter-text">
+                  <h4 class="text-center">NEWSLETTER</h4>
+                    <br />
+                  <div class="text-center" id="newsletter-text-body">
+                      New to Reaction?
+                      Subscribe to our newsletter to receive special offers and latest news.
+                      <br>
+                  </div>
+                  <br>
+                  <div class="input-group">
+                    <input type="text" class="form-control" placeholder="Email here..." />
+                      <span class="input-group-btn">
+                        <button class="btn btn-warning" type="button">Subscribe</button>
+                      </span>
+                  </div>
+                </div>
+              </div>
+            </div>
+            <div class="col-md-4 col-lg-4 container">
+							<h4 class="text-center">JOIN US ON</h4>
+							<div class="row container">
+								<div class="col-md-2">Twitter</div>
+								<div class="col-md-2">Facebook</div>
+							</div>
+                <br>
+                <div class="contact-address">
+                    <address class="text-center">
+                        <strong>Annie-hall RC, Inc.</strong><br>
+                        235 Ikorodu Road,<br>
+                        Ilupeju, Lagos<br>
+                        <abbr title="Phone">P:</abbr> (+123) 456-7890
+                      </address>
+                      
+                      <address class="text-center">
+                        <strong>Admin:</strong><br>
+                        <a href="mailto:#" style="color: white">admin@localhost.com</a>
+                      </address>
+                </div>
+              </div>
+        </div>
+			</div>  
+			<br>
+			<div class="container">
+				<div class="text-center">
+					<span class=text-center>Copyright Annie-hall RC</span>
+					<br>
+					<span class="text-center">2018</span>
+				</div>
+			</div>
     </div>
-  </nav>
-</template>
+  </template>
+    <!-- <nav class="navbar-bottom" role="navigation">
```

Comment:
Instead of commenting, delete it. Unless it will be used in future implementations

Reply:
fixed